test/vlq-tests.ts

```diff
@@ -109,29 +109,27 @@ describe('test encode', function() {
   it('encoder', function() {
     // (lines + segemnts * 6) = byte_count
     let decoded = {
-      mappings: {
```

Comment:
nope, it's dropping the outer `mappings` property since `MappingsEncoder.encode()` no longer expects that. it only needs the mappings so there is no need for passing it in a wrapped form or as a full sourcemap. (see https://github.com/krisselden/fast-source-map/pull/14/commits/f3dd55cefec17766148af780c6391e8d1a0fac2b)